modules/gcp-gke/network.tf

```diff
@@ -9,22 +9,24 @@ data "google_container_cluster" "primary" {
   ]
 }
 
-resource "google_compute_firewall" "gke_private_cluster_istio_gatekeeper_rules" { #tfsec:ignore:google-compute-no-public-ingress
+resource "google_compute_firewall" "gke_private_cluster_cp_to_nodepool_comm" { #tfsec:ignore:google-compute-no-public-ingress
```

Comment:
Can we please use `control_plane` or `master` rather than the `cp` acronym..  `the -comm` suffix is redundant I think.

Reply:
used master in the TF resource name and dropped the -comm suffix